src/modules/edit.js

```diff
@@ -1,39 +1,11 @@
-export const savedTasks = JSON.parse(localStorage.getItem('tasks')) || [];
+import { saveTasks, savedTasks } from './localStorage.js';
 
-export function saveTasks() {
```

Comment:
**Dear @SAZZAD1Q2, at this point, the user is expected to be able to mark task completion by selecting the corresponding checkbox. The updated tasks list should be stored in local storage. All the elements of the user interface should be fully functional and the application should be completed at this stage.**
> In your case 
- [ ] Your task is not loaded from the local storage when the app starts. 
- [ ] checkbox changes are not getting saved to the local storage. 
- [ ] `clear all completed` button is not working 
**Please fix this issues before the next review request**